src/Illuminate/Validation/PresenceVerifierInterface.php

```diff
@@ -4,6 +4,19 @@
 
 interface PresenceVerifierInterface
 {
+    /**
+     * Check for existence of an object in a collection having the given value.
+     *
+     * @param  string  $collection
+     * @param  string  $column
+     * @param  string  $value
+     * @param  int|null  $excludeId
+     * @param  string|null  $idColumn
+     * @param  array  $extra
+     * @return bool
+     */
+    public function getExistence($collection, $column, $value, $excludeId = null, $idColumn = null, array $extra = []);
```

Comment:
It may be possible to make this PR available for `11.x` using docblock instead of actual contract:

```php
/**
 * @method bool getExistence(string $collection, string $column, string $value, int|null $excludeId, string|null $idColumn, array $extra)
 */
interface PresenceVerifierInterface
```